app/lib/actions/scc_manager/sync_products.rb

```diff
@@ -24,6 +24,7 @@ def run
           ::Foreman::Logging.logger('foreman_scc_manager').error "Error while syncronizing SCC-Products: #{e}"
           output[:error] = e.to_s
           output[:status] = 'FAILURE'
```

Comment:
Maybe these two are obsolete then, as well as the evaluation of these variables in `app/lib/actions/scc_manager/sync.rb` by `finalize()`.

Reply:
I thought so, too, but when I tried, I got something like this:
![image](https://user-images.githubusercontent.com/71487468/107003095-863a9280-678c-11eb-9835-b5bc249aa29a.png)

Reply:
I think we should be able to get rid of the `status`-field in `output` completely, since it's only reason for being there is to detect in Sync-action's `finalize()` whether SyncRepo or SyncProduct was unsuccessful.
But this can also be done in a separate PR :slightly_smiling_face:

Reply:
Let's postpone it to a different PR, if you don't mind :)